index.html

```diff
@@ -14,30 +14,266 @@
+                      <div class="title-container">
+                        <h4 class="description-title">Multi-Post Stories</h4>
+                      </div>
+                      <div class="subtitle-container">
+                        <ul class="subtitle-items">
+                          <li class="list-less"><p>CANOPY</p></li>
+                          <li class="list"><p>Back End Dev</p></li>
+                          <li class="list"><p>2015</p></li>
+                        </ul>
+                      </div>
+                      <div class="article-container">
+                        <p class="description-item">
+                          A daily selection of privately perosonalized reads; no accounts
+                          or sign-ups required.
+                        </p>
+                      </div>
+                      <div class="tag-container">
+                        <ul class="tag-item">
+                          <li class="tag-list-item"><a class="tag-btn" href="#">html</a></li>
+                          <li class="tag-list-item"><a class="tag-btn" href="#">css</a></li>
+                          <li class="tag-list-item"><a class="tag-btn" href="#">javaScript</a></li>
+                        </ul>
+                      </div>
+                      <div class="buttom-container">
+                        <input type="button" value="See Project" class="project-buttom" />
+                      </div>
+                    </div>
+                </div>
+            </div>
+            <div class="sample-project project-3">
+                <div class="project-header ">
+                    <div class="image-holder bg">
+                        <img src="./Assets/SnapshootPortfolio2.png" alt="header" class="header-img"/>
+                    </div>
+
+                </div>
+                <div class="project-content">
+                    <div class="project-description">
+                      <div class="title-container">
+                        <h4 class="description-title">Tonic</h4>
+                      </div>
+                      <div class="subtitle-container">
+                        <ul class="subtitle-items">
+                          <li class="list-less"><p>CANOPY</p></li>
+                          <li class="list"><p>Back End Dev</p></li>
+                          <li class="list"><p>2015</p></li>
+                        </ul>
+                      </div>
+                      <div class="article-container">
+                        <p class="description-item">
+                          A daily selection of privately perosonalized reads; no accounts
+                          or sign-ups required.
+                        </p>
+                      </div>
+                      <div class="tag-container">
+                        <ul class="tag-item">
+                          <li class="tag-list-item"><a class="tag-btn" href="#">html</a></li>
```

Comment:
[OPTIONAL]
- [ ] Please consider removing the `a` tag as this is a normal text and not a link